crates/sui-types/src/sui_system_state.rs

```diff
@@ -374,7 +375,8 @@ pub struct ValidatorSet {
 /// Rust version of the Move sui::sui_system::SuiSystemStateInner type
 /// We want to keep it named as SuiSystemState in Rust since this is the primary interface type.
 #[derive(Debug, Serialize, Deserialize, Clone, Eq, PartialEq, JsonSchema)]
-pub struct SuiSystemState {
+#[serde(rename = "SuiSystemState")]
```

Comment:
renaming this to `SuiSystemState` will cause json schema clash with the `SuiSystemState` object, it's better to keep the original name.